spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/ConfigHttpClient.java

```diff
@@ -148,4 +172,38 @@ private static String copyInputStream(InputStream inputStream) throws IOExceptio
             }
         }
     }
+    
+    /**
+     * Checks if Azure App Configuration Tracing is disabled, and if not gets tracing
+     * information.
+     * 
+     * @param request The http request that will be traced, used to check operation being run.
+     * @return String of the value for the correlation-context header. 
+     */
+    private static String getTracingInfo(HttpUriRequest request) {
+        String track = System.getenv(RequestTracingConstants.AZURE_APP_CONFIGURATION_TRACING_DISABLED.toString());
+        if (track != null && track.equalsIgnoreCase("false")) {
+            return "";
+        }
+        String requestTypeValue = request.getURI().getPath().startsWith("/kv") ? RequestType.STARTUP.toString()
```

Comment:
nit: for several adjacent lines of code, how about adding empty lines between lines to split the blocks for better readability.